pages/documentation/showcase.md

```diff
@@ -48,138 +52,136 @@ If your project is currently using USWDS and you would like to see it included i
 - [Supreme Court of the United States](https://www.supremecourt.gov/)
+- [Tech at GSA](https://tech.gsa.gov/)
 - [TechFAR Hub](https://techfarhub.cio.gov/)
 - [Technology Modernization Fund](https://tmf.cio.gov/)
-- [Thrift Savings Plan (TSP)](https://www.tsp.gov/)
+- [Thrift Savings Plan](https://www.tsp.gov/)
 - [Touchpoints](https://touchpoints.digital.gov/)
-- [Transportation Security Administration (TSA)](https://www.tsa.gov/)
+- [Transportation Security Administration](https://www.tsa.gov/)
 - [U.S. Access Board](https://www.access-board.gov)
 - [U.S. Agency for Global Media](https://www.usagm.gov/)
-- [U.S. Agency for International Development (USAID)](https://www.usaid.gov/)
-- [U.S. Army Medical Center of Excellence (MEDCoE)](https://medcoe.army.mil)
+- [U.S. Agency for International Development](https://www.usaid.gov/)
+- [U.S. Army Medical Center of Excellence](https://medcoe.army.mil)
+- [U.S. Citizenship and Immigration Services](https://www.uscis.gov/)
+- [U.S. Courts](http://www.uscourts.gov/)
 - [U.S. Data Federation](http://federation.data.gov/)
-- [U.S. Department of Agriculture (USDA)](https://www.usda.gov/)
-- [U.S. Department of Justice - Civil Rights Division](https://civilrights.justice.gov/)
+- [U.S. Department of Agriculture](https://www.usda.gov/)
 - [U.S. Department of Labor](https://www.dol.gov/)
 - [U.S. Department of the Treasury](https://treasury.gov)
 - [U.S. Department of Veterans Affairs](https://va.gov)
 - [U.S. Digital Service](https://www.usds.gov/)
 - [U.S. Emerging Citizen Technology Atlas](https://emerging.digital.gov/)
+- [U.S. Environmental Protection Agency](https://www.epa.gov/)
 - [U.S. Federal Labor Relations Authority](https://www.flra.gov/)
+- [U.S. Geological Survey](https://www.usgs.gov/)
 - [U.S. Indicators For The Sustainable Development Goals](https://sdg.data.gov/)
+- [U.S. Interagency Council on Homelessness](https://www.usich.gov/)
+- [U.S. Kimberley Process Authority](https://www.uskpa.org/)
+- [U.S. Sentencing Commission Guidelines Manual](https://guidelines.ussc.gov/)
 - [Unified Shared Services Management](https://www.ussm.gov/)
-- [United States Citizenship and Immigration Services (USCIS)](https://www.uscis.gov/)
-- [United States Courts](http://www.uscourts.gov/)
-- [United States Interagency Council on Homelessness](https://www.usich.gov/)
-- [United States Sentencing Commission Guidelines Manual](https://guidelines.ussc.gov/)
 - [Unlocking Federal Talent](https://unlocktalent.gov/)
-- [United States Kimberley Process Authority (USKPA)](https://www.uskpa.org/)
 - [USA.gov](https://usa.gov)
-- [USAID](https://www.usaid.gov/)
 - [USAJOBS](https://www.usajobs.gov/)
 - [USAJOBS Data Science Careers](https://www.usajobs.gov/careerfields/data-science)
 - [USAJOBS Design System](http://usajobs.github.io/design-system/)
 - [USAspending.gov](https://www.usaspending.gov/)
-- [USDA Economic Research Service](http://www.ers.usda.gov/)
 - [USDA New Farmers](https://newfarmers.usda.gov/)
```

Comment:
Should this be moved to maintain alphabetical order?

Reply:
@finekatie @sarah-sch I updated the order of this in [9ec848c](https://github.com/uswds/uswds-site/pull/2272/commits/9ec848c8587323c5b16f5a8c45bed06395f303bc). Let me know if you need any changes!

Reply:
I would leave this.